app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -17,6 +18,37 @@
   color: govuk-colour("white");
 }
 
-.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
+.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:after {
   border-color: govuk-colour("white");
 }
+
+.govuk-breadcrumbs__list-item {
+  margin-left: 0;
+  padding-left: 0;
+  margin-right: 20px;
+  @include chevron($direction: "right", $color: govuk-colour("dark-grey"));
+
+  &:before,
+  &:last-of-type:after {
+    display: none;
+  }
+
+  @include govuk-media-query($until: tablet) {
+    display: none;
+    margin-bottom: 0;
+    padding: 14px 0;
+
+    &:last-of-type {
+      display: inline-block;
+      padding-left: 0;
+      margin-right: 0;
+      max-width: 72%;
+    }
+
+    &:first-of-type {
+      display: inline-block;
+      padding-left: 0;
+      max-width: 22%;
```

Comment:
Sorry, but this still doesn't work properly when the text in the first breadcrumb is something other than 'Home'.

![Screenshot 2020-02-12 at 11 18 47](https://user-images.githubusercontent.com/861310/74330579-d2294600-4d89-11ea-80c7-208d24543b42.png)

You might have to use absolute positioning on the chevron to crack this.

Reply:
have tweaked again...

Reply:
Seems to be better now.